.github/workflows/python-package.yml

```diff
@@ -4,42 +4,27 @@ on:
   push:
```

Comment:
This is just a general comment. I think we might want to check that GH action workflows are consistent

Reply:
Tbh, virtually every element of the biocommons template is inconsistent across the repos... not really sure if the best principle is to try to standardize all at once or move iteratively.